docs/UserGuide.md

```diff
@@ -3,91 +3,128 @@
-
-### 1.3 Product Overview
+### 1.2 Product Overview
 
 ra.VI is targeted at the NUS freshman. As a freshman, there are many documents and new procedures that you must get familiar with. This may be daunting for you but ra.VI will provide you with the assistance you need. 
 By helping you keep track of your tasks and deadlines, you will be able to keep on top of deadlines. 
 Moreover, the timetable feature helps you to schedule your lessons, allowing you to be more prepared for lessons.
 Adding on, ra.VI also allows you to keep track of your CAP, so that you can keep tabs on how well you are doing, motivating you towards that elusive CAP 5.0.
 
-### 1.4 Quick Start
+### 1.3 Keywords
+
+**Tasks**
+
+A task refers to something that you would like to get done. This refers to reading a book or doing an assignment. Tasks can be created with or without a deadline. <br>
+
+**Deadline**
+
+A deadline refers to a date and time associated with a task. This is in the format: DAY-MONTH-YEAR TIME_24H, e.g. `23-12-2020 1400` or `1-2-2000 0800` <br>
+
+**Modules**
+
+A module refers to a module taken under NUS. Entered modules will be checked against the list of NUS modules. <br>
+
+**CAP**
+
+CAP refers to Cumulative Average Point, which is the grading system used by NUS. CAP will be calculated based on your modules. <br>
+
+**Done**
+
+A task can be marked as done. This will signify completion of the task so that you can get an easy view of any remaining tasks. <br>
+
+**Timetable**
+
+The timetable is specific to you. It allows you to add lessons to your timetable with respect to the modules that you are taking. After setting the timetable up the first time, you do not need to go through the set up again. <br>
+
+**Day**
+
+Values for `<day>`:\
+`MONDAY`, `TUESDAY`, `WEDNESDAY`, `THURSDAY`, `FRIDAY`, `SATURDAY`, `SUNDAY`
+
+**Lesson Type**
+
+Values for `<lesson type>`:\
+`TUTORIAL`, `LECTURE`, `SEMINAR`, `LAB`, `RECITATION`, `SESSION`
+
+**Time**
+
+Format of `<time>`:\
+Must be in the 24h format e.g. `0900`, `1415`
+
+### 2 Quick Start
 
 1. Ensure that you have Java 11 or above installed.
-2. Download the latest version of `ra.VI` from [here](https://github.com/AY2021S1-CS2113T-T09-2/tp/releases/tag/v1.0).
-3. Copy the file to the folder you want to use as the home folder for your scheduler. Double-click the file to start the app. 
-4. Type the command in the command box and press Enter to execute it. e.g. typing help and pressing Enter will open the help window.  
+2. Download the latest version of `ra.VI` from [here](https://github.com/AY2021S1-CS2113T-T09-2/tp/releases/tag/v2.0).
+3. Copy the file to the folder you want to use as the home folder for `ra.VI`.
+4. Open a command prompt in the folder from step 3 and enter `java -jar ravi.jar`.
```

Comment:
Not a comment per se but I'm making a shell script (one for Windows, one for Unix) to check Java version and run ra.VI so this might need to be updated next time.

Reply:
Noted